projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -329,6 +329,7 @@ const Article = ({
 				onMessage={(event) => {
 					handlePing(event.nativeEvent.data);
 				}}
+				decelerationRate={'normal'}
```

Comment:
https://github.com/react-native-webview/react-native-webview/blob/30685edda045f7ba5d1400b415278e1109436dcd/docs/Reference.md#decelerationrate

Looks like ios feature only, shall we apply `if ios then...` logic?

Reply:
We can do, I'm not sure if it's necessary though. My understanding was that separation of platform specific props is handled by the package so OS props would be ignored by Android  and vice versa but maybe I'm wrong here? It looks like a fair few of the other props we are using are also platform specific without any platform `if` logic.